packages/test-runner-commands/test/send-keys/sendKeysPlugin.test.ts

```diff
@@ -0,0 +1,35 @@
+import path from 'path';
+import { platform } from 'os';
+import { runTests } from '@web/test-runner-core/test-helpers';
+import { chromeLauncher } from '@web/test-runner-chrome';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+
+import { sendKeysPlugin } from '../../src/sendKeysPlugin';
+
+describe('sendKeysPlugin', function test() {
+  this.timeout(20000);
+
+  it('can set the viewport on puppeteer', async () => {
+    await runTests({
+      files: [path.join(__dirname, 'browser-test.js')],
+      browsers: [chromeLauncher()],
+      plugins: [sendKeysPlugin()],
+    });
+  });
+
+  // playwright doesn't work on windows VM right now
+  if (platform() !== 'win32') {
+    it('can set the viewport on playwright', async () => {
+      await runTests({
+        files: [path.join(__dirname, 'browser-test.js')],
+        browsers: [
+          playwrightLauncher({ product: 'chromium' }),
+          playwrightLauncher({ product: 'firefox' }),
+          // TODO: make webkit work in the CI
+          // playwrightLauncher({ product: 'webkit' }),
```

Comment:
Is this TODO copied from another test? If so can we check if works now?

Reply:
It is indeed. I'll switch it on and see what happens...

Reply:
That seems to have run...should I switch that on in other tests?

Reply:
Yeah!